### PR TITLE
DropDownButton: Add aria-label to the List

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js [hunk 1]
```diff
@@ -2672,4 +2673,21 @@ QUnit.module('Accessibility', {
...
+
+            assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'No data to display');
+
+            instance.option(dataSource, [1, 2, 3]);
+            assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'List');
+
+            instance.option(dataSource, []);
+            assert.strictEqual($(`.${LIST_CLASS}`).attr('aria-label'), 'No data to display');
+        });
+    });
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2720]: what about runtime change?

Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2720]: This test has checked runtime dataSource change.

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2720]: What about behavior on initialization when items are not empty? :)

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2692]: `items` option change is not tested also

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2720]: please, add tests for all scenarios

Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:2720]: Tests were added here: [faa13d2c571fc10b7e709bb118c88a5413822707].

**Review by ksercs — APPROVED** [review]

**Review by Krijovnick — APPROVED** [review]

**Pull request merged**